index.src.html

```diff
@@ -890,6 +893,24 @@ <h3 id="user-controls">User Controls</h3>
   active <a>referrer policy</a> on a page.
 </section>
 
+<section>
+  <h2 id="authoring">Authoring Considerations</h2>
+
+  <h3 id="unknown-policy-values">Unknown Policy Values</h3>
+
+  As described in <a section href="#determine-policy-for-token"></a>
```

Comment:
Nit: Sections can be written `[[#determine-policy-for-token]]`, which is shorter. :)